addons_marketplace/marketplace-manager/api/v1/schema.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+from marshmallow import Schema, fields
+
+
+class StatusEnum(Enum):
+    UNDER_REVIEW = "under_review"
+    APPROVED = "approved"
+
+
+class ServiceSchema(Schema):
+    service_name = fields.String(required=True)
+    image_uri = fields.String(required=True)
+    command = fields.String()
+    networks = fields.List(fields.String(), default=[])
+    ports = fields.Dict(keys=fields.String(), values=fields.String(), default={})
+    environment = fields.Dict(keys=fields.String(), values=fields.String(), default={})
+    labels = fields.Dict(keys=fields.String(), values=fields.String(), default={})
+
+
+class MarketplaceAddonSchema(Schema):
+    _id = fields.String()
+    name = fields.String(required=True)
+    description = fields.String()
+    services = fields.Nested(ServiceSchema, many=True, required=True)
```

Comment:
What exactly are services in the context of marketplace addons?

Reply:
similar to docker-compose, the services keyword indicates the containers.

Reply:
If you are fine with this that its ok. 
(For me who is not swimming in your topic I instantly associate this with oakestra services, so a simply change to addon_services would be cool, but up to you. I am fine with how it is now)